src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private final Random randomColor = new Random();
```

Comment:
The variable 'randomColor' is misleading. It's not a color, it's a Random object. Consider renaming it to 'random'.

Reply:
done